libmcount/mcount.c

```diff
@@ -530,14 +530,6 @@ void mcount_entry_filter_record(struct mcount_thread_data *mtdp,
 				save_argument(mtdp, rstack, tr->pargs, regs);
 			if (tr->flags & TRIGGER_FL_READ)
 				save_trigger_read(mtdp, rstack, tr->read);
-
-			if (mtdp->nr_events) {
-				/*
-				 * Flush rstacks if event was recorded as it only has
-				 * limited space for the events.
-				 */
-				record_trace_data(mtdp, rstack, NULL);
-			}
```

Comment:
For the reason removing the above lines,
what about additionally appending comments ?  like below:

> The commit 9321fc5 ("mcount: Implement reading memory stat on trigger") flush rstacks
> if event was recorded at the entry but let it can work at the exit for time-filter

I'm not sure this comment is best but I thought we can add `The commit 9321fc5 ("mcount: Implement reading memory stat on trigger") ` to commit log message.

Reply:
@Taeung I think it's better for me to have such comment. Thanks!